tpm2/src/util/tpm2_kmyth_seal.c

```diff
@@ -30,12 +30,443 @@ extern const cipher_t cipher_list[];
 //############################################################################
 // tpm2_kmyth_seal()
 //############################################################################
-int tpm2_kmyth_seal(char *input_path,
-                    char *output_path,
+int tpm2_kmyth_seal(uint8_t * input, size_t input_length,
+                    uint8_t ** output, size_t *output_length,
                     char *auth_string,
                     char *pcrs_string,
```

Comment:
Not for this pull request, but what sort of things can go in `auth_string`? Is there any reason it should be able to contain a `NULL` byte?